mm-count-associated-devices.ash

```diff
@@ -0,0 +1,23 @@
+#!/bin/ash
```

Comment:
echo $SHELL outputs /bin/ash on some devices, and previous bash shell scripts were not compatible. I suppose .sh is sufficient, but I had initially used .ash. Should I change to .sh?